app/controllers/accounts_controller.rb

```diff
@@ -82,7 +82,11 @@ def sync
   private
 
   def set_account
-    @account = Current.family.accounts.find(params[:id])
+    begin
+      @account = Current.family.accounts.find(params[:id])
```

Comment:
In Rails this error handling is typically not necessary as the `@account` would just be `nil` if not found.

Reply:
Yeah, this was actually a fix for something else. If you navigate to an `account_id` that doesn't exist, rails spits out stack traces which is not very graceful.